### PR TITLE
Arrays in POST body should not be json encoded to strings

### PR DESCRIPTION
**Pull request by baumatron**
POST body parameters should not be json encoded by a filter, which converts the array to a json encoded string.

### CODE REVIEW

**Comment by tbarn** [comment]
The previous query with a double `group_by` that resulted in:  
```array(1) { ["result"]=> array(1) { [0]=> array(2) { ["["item.title","item.total"]"]=> NULL ["result"]=> int(416) } } } Array``` 

Now has the correct result with all `group_by`'s

👍 Thanks for working on this, @baumatron!

File: src/Client/Resources/keen-io-3_0.php [hunk 1]
```diff
@@ -1235,7 +1216,6 @@
                     'description' => 'A Timeframe specifies the events to use for analysis based on a window of time. '
                                     . ' If no timeframe is specified, all events will be counted.',
                     'type'        => array('string', 'array'),
```
Comment (masojus) [inline src/Client/Resources/keen-io-3_0.php]: Did you test to make sure it still works with either a single string, or an array of 1+ strings?

Comment (masojus) [reply src/Client/Resources/keen-io-3_0.php]: Same comment for `group_by` where applicable, of course.

Comment (baumatron) [reply src/Client/Resources/keen-io-3_0.php]: Yeah, I tested all possible combinations and they work as long as they're actually valid timeframes. That includes a relative timeframe, which would be a string, as well as an "array", which is more like a dictionary, containing an absolute timeframe. More than one relative timeframe returns an error.

Comment (baumatron) [reply src/Client/Resources/keen-io-3_0.php]: `group_by` also works as expected with a single string or an array of strings.

**Comment by baumatron** [comment]
Yeah, that description seems off. Not related to this change though so that can be a follow up.

**Comment by masojus** [comment]
Yeah feel free to file an issue to follow up and fix the descriptions. Looks good to me otherwise.

**Comment by baumatron** [comment]
#124

**Comment by baumatron** [comment]
I think this looks good to merge once someone with write access gives it approval.

**Pull request merged**